evalml/pipelines/pipeline_base.py

```diff
@@ -222,6 +223,7 @@ def describe(self, return_dict=False):
         Returns:
             dict: Dictionary of all component parameters if return_dict is True, else None
         """
+        logger = get_logger(f"{__name__}.describe")
```

Comment:
Why do this? We're already getting the logger using `logging.getLogger(__name__)`